site/tests/conftest.py

```diff
@@ -82,5 +111,341 @@ def client_with_login(client, users):
     """Log in a user to the client."""
     user = users[0]
     login_user(user)
-    login_user_via_session(client, email=user.email)
+    login_user_via_session(client, user)
     return client
+
+
+@pytest.fixture(scope="module")
+def resource_type_v(app):
```

Comment:
Many of these will conflict with https://github.com/zenodo/zenodo-rdm/pull/352 but we'll fix it when the time comes